components/patient-data/ui/src/main/resources/PhenoTips/PatientClass.xml

```diff
@@ -257,6 +257,19 @@
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
     </date_of_death_entered>
+    <delivery_mode>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <name>delivery_mode</name>
+      <number>47</number>
+      <picker>0</picker>
+      <prettyName>Delivery mode</prettyName>
+      <size>30</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
```

Comment:
@sdumitriu @bgallinger - Yes how about these options for the "Mode of Delivery" list:
- NSVD
- VAD
- Forceps delivery
- Odon device
- Primary caesarean section
- Secondary caesarean section
- VBAC

I have requested NSVD, primary and secondary C-section, and VBAC . VAD is already a term, under "ventouse"